eng/pipelines/core-splitting.yml

```
@@ -0,0 +1,75 @@
+parameters:
+- name: OutputFolder
+  type: string
+  default: "projects"
+- name: numOfTestsPreJob
+  type: number
+  default: 20
+steps:
+  - pwsh: |
+      dotnet build /t:ProjectDependsOn ./eng/service.proj `
+        /p:ServiceDirectory=* /p:TestTargetDependency="Azure.Core" `
```

```suggestion
        /p:ServiceDirectory=* 
        /p:ProjectDependsOnDependency="Azure.Core" `
```

this is just clean-up to make the names align with the target we are using. We should also make sure we are thinking about making Azure.Core a parameter to this template so we can use it in other pipelines as well.